app/views/backend/fixed_assets/_form.html.haml

```diff
@@ -4,6 +4,7 @@
   = f.input :name
   = f.input :description, as: :text
   = f.referenced_association :product, :source => :depreciables
+  = f.association :creator, input_html: { value: current_user.id }, as: :hidden
```

Comment:
Risky. If we need this, it needs to be moved up into the controller.
Also: don't timestamps already handle this?